diff --git a/package_upload.py b/package_upload.py
index b94b3d0..64f6e9e 100755
--- a/package_upload.py
+++ b/package_upload.py
@@ -22,10 +22,9 @@ def get_current_branch():
 
 
 if __name__ == '__main__':
-    dt = datetime.datetime.now()
-    msg = dt.strftime("%Y%m%d%H%M")
+    msg = raw_input("请输入commit描述信息: ")
     current_branch = get_current_branch()
 
     run_cmd("git add *")
-    run_cmd("git commit -a -m {}".format(msg))
+    run_cmd("git commit -a -m '{}'".format(msg))
     run_cmd("git push origin {}".format(current_branch))
\ No newline at end of file
